vsphere/datadog_checks/vsphere/data/conf.yaml.example

```diff
@@ -99,24 +99,22 @@ instances:
   #  include_only_marked: false
 
   ## @param all_metrics - boolean - optional - default: false
-  ## When set to true, this check collects EVERY metric
-  ## from vCenter, which means a LOT of metrics you probably
-  ## do not care about. We have selected a set of metrics
-  ## that are interesting to monitor for you if false
-  ## This parameter will be deprecated in favor of `collection_level`
+  ## This parameter has been deprecated in favor of `collection_level`
+  ## Please set `collection_level: 4` if you want to collect all the metrics
+  ## available on your system.
   #
   #  all_metrics: false
 
   ## @param collection_level - integer - optional - default: 1
   ## A number between 1 and 4 to specify which metrics will be sent
```

Comment:
if we remove the sentence below (see other comment) we can s/which metrics/how many metrics